web/src/utils/pkt-drop.ts

```diff
@@ -219,8 +345,18 @@ export type DROP_CAUSES_VALUES = typeof dropCausesValues[number];
 const dropCausesNames = DROP_CAUSES.map(v => v.name);
 export type DROP_CAUSES_NAMES = typeof dropCausesNames[number];
 
-export const DROP_CAUSES_DOC_URL = 'https://github.com/torvalds/linux/blob/master/include/net/dropreason-core.h';
+export const CORE_DROP_CAUSES_DOC_URL = 'https://github.com/torvalds/linux/blob/master/include/net/dropreason-core.h';
+export const OVS_DROP_CAUSES_DOC_URL =
+  'https://git.kernel.org/pub/scm/linux/kernel/git/netdev/net-next.git/tree/net/openvswitch/drop.h';
 
 export const getDropCauseDescription = (name: DROP_CAUSES_NAMES): string => {
   return DROP_CAUSES.find(v => v.name === name)?.description || 'Unknown';
 };
+
+export const getDropCauseDocUrl = (name: DROP_CAUSES_NAMES): string => {
+  if (name.includes('OVS')) {
```

Comment:
```suggestion
  if (name.startsWith('OVS_')) {
```

nit: would be safer to check start prefix including `_`